jvm-packages/dev/change_version.sh

```diff
@@ -1,27 +1,19 @@
-#!/bin/sh
-
-#!/bin/bash
-
+#!/usr/bin/env bash
 #
-# Licensed to the Apache Software Foundation (ASF) under one or more
-# contributor license agreements.  See the NOTICE file distributed with
-# this work for additional information regarding copyright ownership.
-# The ASF licenses this file to You under the Apache License, Version 2.0
-# (the "License"); you may not use this file except in compliance with
-# the License.  You may obtain a copy of the License at
+# Copyright (c) 2019 by Contributors
```

Comment:
don't add years to avoid keep changing it

Reply:
I am fairly sure you need to keep the year of release for legal reasons.

Reply:
Just look at the code base how many 2014 2017 2018 have been there

Reply:
When I look through the project I see lots of examples where they have put the year with copyright deceleration.
* https://github.com/dmlc/xgboost/blob/master/jvm-packages/xgboost4j/src/main/java/ml/dmlc/xgboost4j/java/DMatrix.java#L2
* https://github.com/dmlc/xgboost/blob/master/jvm-packages/xgboost4j-spark/src/main/scala/ml/dmlc/xgboost4j/scala/spark/XGBoostRegressor.scala#L2

Its not safe to remove the copyright deceleration.

Reply:
those should be removed, I was not aware that our scala style checker is actually a bit off

https://github.com/dmlc/xgboost/blob/master/jvm-packages/scalastyle-config.xml#L52 this should be removed and we should follow standard apache license header which does not include year info
https://www.apache.org/legal/src-headers.html#headers